triangle.json

```diff
@@ -0,0 +1,98 @@
+{
+    "#": [
+        "Pursuant to discussion in #202,",
+        "we have decided NOT to test triangles where a + b = c.",
+        "e.g: (2, 4, 2, Isosceles), (1, 3, 4, Scalene).",
+        "It's true that the triangle inequality admits such triangles.",
+        "These triangles have zero area, however.",
+
+        "The expectation are given as strings,",
+        "but your language may use the appropriate representations.",
+        "For example, enums, variants, or tagged unions all are viable candidates.",
+
+        "Your track may choose to have the 'illegal' result be another member of the enum/variant/union/etc.,",
+        "or instead to signal an error/exception/etc. on an illegal triangle.",
+
+        "If appropriate for your track, you'll need to ensure that no pair of expected values are equal.",
+        "Otherwise, an implementation that always returns a constant value may falsely pass the tests.",
+        "See https://github.com/exercism/xgo/pull/208"
```

Comment:
also applicable to sublist because https://github.com/exercism/xpython/issues/342 - maybe I'll add a comment to that json file